.prod/uwsgi.ini

```diff
@@ -9,3 +9,9 @@ gid = appuser
 master = 1
 processes = 2
 threads = 2
+cron = -1 -1 -1 -1 -1 /app/manage.py send_mail
```

Comment:
this is probably fine for now, when the service does not have that much traffic, but what if in the future sending all emails would take longer than 1 munite? 🤔 would every 5 minutes be a good enough frequency? or I am overthinking this? sending emails every minutes might be better from the UX POV for sure. 🙂

Reply:
The thing is we also send notification for signup user and enrolment notification. I don't think 5 minutes delay is idea for such kinds of notification. I've tested with 1k emails and it tooks about 10-15 seconds so I won't think this should be a problem for now,  unless we have a huge email template

Reply:
ok 👍 let's go with the "every minute".